public-ui/src/app/utils/spinner.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+
+export const spinnerCss = `
+.spinner-container {
+    width:100%;
+    display: flex;
+    height: auto;
+    align-content: center;
+    align-items: center;
+    justify-content: center;
+}
+
+.loading-dots {
+    margin-top:.25rem;
+    width:2rem;
+    display:flex;
+    height:auto;
+    justify-content:space-between;
+  }
+  .loading-dots .dot {
+    width:.25rem;
+    height:.25rem;
+    background:transparent;
+    border-radius: 50%;
+    animation:load 1s linear infinite alternate;
+  }
+  .loading-dots .dot:first-of-type{
+    animation-delay: .25s;
+  }
+  .loading-dots .dot:nth-of-type(2){
+    animation-delay: .5s;
+  }
+  .loading-dots .dot:nth-of-type(3){
+    animation-delay: .75s;
+  }
+  .loading-dots .dot:nth-of-type(4){
+    animation-delay: 1s;
+  }
+  a:link,a:visited,a{
+    color:#2aa3d8;
+  }
+  @keyframes load {
+    from{background:transparent}
+    to{background:#302c70}
+  }
+`;
+
+
+interface SpinnerProps {
+  loading: boolean;
+  dots: boolean;
+}
```

Comment:
I think we can remove both props. `loading` should be handled in the parent and only render the spinner if true. Also, we should split the two types into separate consts. 

So once we convert the parent components, should look something like this:
```
{loading && <Spinner/>}
{loading && <LoadingDots/>} // or whatever we choose to name this one
```

Reply:
I split these both into different consts.